react/flagMap.tsx

```diff
@@ -0,0 +1,235 @@
+  RCA: React.lazy(() => import('./icons/RCA')),
+  ROU: React.lazy(() => import('./icons/ROU')),
+  RUS: React.lazy(() => import('./icons/RUS')),
+  RWA: React.lazy(() => import('./icons/RWA')),
+  SAU: React.lazy(() => import('./icons/SAU')),
+  SDN: React.lazy(() => import('./icons/SDN')),
+  SEN: React.lazy(() => import('./icons/SEN')),
+  SGP: React.lazy(() => import('./icons/SGP')),
+  SGS: React.lazy(() => import('./icons/SGS')),
+  SLB: React.lazy(() => import('./icons/SLB')),
+  SLE: React.lazy(() => import('./icons/SLE')),
+  SLV: React.lazy(() => import('./icons/SLV')),
+  SMR: React.lazy(() => import('./icons/SMR')),
+  SOM: React.lazy(() => import('./icons/SOM')),
+  SRB: React.lazy(() => import('./icons/SRB')),
+  SSD: React.lazy(() => import('./icons/SSD')),
+  STP: React.lazy(() => import('./icons/STP')),
+  SUR: React.lazy(() => import('./icons/SUR')),
+  SVK: React.lazy(() => import('./icons/SVK')),
+  SVN: React.lazy(() => import('./icons/SVN')),
+  SWE: React.lazy(() => import('./icons/SWE')),
+  SWZ: React.lazy(() => import('./icons/SWZ')),
+  SYC: React.lazy(() => import('./icons/SYC')),
+  SYR: React.lazy(() => import('./icons/SYR')),
+  TCD: React.lazy(() => import('./icons/TCD')),
+  TGO: React.lazy(() => import('./icons/TGO')),
+  THA: React.lazy(() => import('./icons/THA')),
+  TJK: React.lazy(() => import('./icons/TJK')),
+  TKM: React.lazy(() => import('./icons/TKM')),
+  TMP: React.lazy(() => import('./icons/TMP')),
+  TON: React.lazy(() => import('./icons/TON')),
+  TTO: React.lazy(() => import('./icons/TTO')),
+  TUN: React.lazy(() => import('./icons/TUN')),
+  TUR: React.lazy(() => import('./icons/TUR')),
+  TUV: React.lazy(() => import('./icons/TUV')),
+  TWN: React.lazy(() => import('./icons/TWN')),
+  TZA: React.lazy(() => import('./icons/TZA')),
+  UGA: React.lazy(() => import('./icons/UGA')),
+  UKR: React.lazy(() => import('./icons/UKR')),
+  UNK: React.lazy(() => import('./icons/UNK')),
+  URY: React.lazy(() => import('./icons/URY')),
+  USA: React.lazy(() => import('./icons/USA')),
+  UZB: React.lazy(() => import('./icons/UZB')),
+  VCT: React.lazy(() => import('./icons/VCT')),
+  VEN: React.lazy(() => import('./icons/VEN')),
+  VGB: React.lazy(() => import('./icons/VGB')),
+  VIR: React.lazy(() => import('./icons/VIR')),
+  VNM: React.lazy(() => import('./icons/VNM')),
+  VUT: React.lazy(() => import('./icons/VUT')),
+  WSM: React.lazy(() => import('./icons/WSM')),
+  YEM: React.lazy(() => import('./icons/YEM')),
+  ZAF: React.lazy(() => import('./icons/ZAF')),
+  ZMB: React.lazy(() => import('./icons/ZMB')),
+  ZWE: React.lazy(() => import('./icons/ZWE')),
+}
+
+export type CountryCode = keyof typeof flagMap
+
+export function loadFlag(countryCode?: CountryCode): React.ReactElement | null {
+  if (!countryCode || !flagMap[countryCode]) {
+    return null
+  }
+
+  if (typeof window === 'undefined') {
+    return <div style={{ height: 24, width: 24 }} />
+  }
+
+  const Flag = flagMap[countryCode]
```

Comment:
```suggestion
  const Flag = flagMap[countryCode]
  if (typeof window === 'undefined' || !Flag) {
    return <div style={{ height: 24, width: 24 }} />
  }
```

Reply:
actually we need to return `null` here if there is no flag component, thanks for the reminder!